src/user/user.controller.ts

```diff
@@ -1,9 +1,18 @@
-import { Controller, Get, Query, Param, ParseIntPipe } from '@nestjs/common';
-import { ApiTags } from '@nestjs/swagger';
+import {
+  Controller,
+  Get,
+  Query,
+  Param,
+  ParseIntPipe,
+  Post,
+  UseGuards,
+} from '@nestjs/common';
+import { ApiCookieAuth, ApiTags } from '@nestjs/swagger';
 import { UserService } from './user.service';
 import { AppValidationPipe } from '../utils/validation.pipe';
 import { UserFindDto } from './dtos/user.find.dto';
 import { UsersResult } from './dtos/user.show.dto';
+import { IsSuperAdminGuard } from '../auth/is.super.admin.guard';
```

Comment:
Should we skip this superadmin user in UserController,get method?

Reply:
done in new changes pushed